types/barnard59-core/package.json

```diff
@@ -0,0 +1,7 @@
+{
+    "private": true,
+    "dependencies": {
+        "rdf-js": "^4.0.2",
```

Comment:
only this little detail, why this dependency? It's not in original package deps list:
https://github.com/zazuko/barnard59-core/blob/master/package.json

Reply:
This is a types-only package, which defines data interfaces used throughout this and related packages.

But you have a point. In actual typings I import from `@rdfjs/types` (one proxies the other). Might actually align that

Reply:
Ah, shoot, it's not in the allowed list. I reverted this for now if that's ok with you.

For the future: https://github.com/microsoft/DefinitelyTyped-tools/pull/310